metals/src/main/scala/scala/meta/internal/metals/BloopServers.scala

```diff
@@ -44,19 +46,30 @@ final class BloopServers(
     if (!result) {
       scribe.warn("There were issues stopping the Bloop server.")
       scribe.warn(
-        "If it doesn't start back up you can run the `build-restart` command manually."
+        "If it doesn't start badefck up you can run the `build-restart` command manually."
```

Comment:
ha, I don't think this was intentional.

Reply:
That's where it went :P I was sure I was writing somewhere else.

Reply:
Fixed